lib/dynamic-help-provider.ts

```diff
@@ -20,7 +20,6 @@ export class DynamicHelpProvider implements IDynamicHelpProvider {
 		return ((): IDictionary<any> => {
 			let isHtml = options.isHtml;
 			let localVariables:IDictionary<any> = {};
-			localVariables["isMobileWebsite"] = isHtml || this.isProjectType([this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.MobileWebsite]).wait();
```

Comment:
there are a lot of
```JavaScript
<% if (isMobileWebsite) ...
```
or some similar ones, which use this variable in our .md files. We should remove the usage from there or keep this variable.